Automattic/woocommerce-payments
# Latest develop merge to POC branch

Another PR to resolve conflicts between `develop` and `poc/upe-instances-multiplied`.

**Conflicts resolved in:**

class-wc-payments-upe-checkout.php
test-class-upe-payment-gateway.php

Conflicts were not difficult to resolve, and nothing should modify the behavior we are working on.

#### Testing instructions
Buy a product using either shortcode or Blocks checkout and ensure there are no errors.

[E2E tests are green.](https://github.com/Automattic/woocommerce-payments/actions/runs/3765808253)




-------------------


**Post merge**
